zebra/interface.c

```diff
@@ -1447,27 +1532,33 @@ static void interface_update_stats(void)
 
 struct cmd_node interface_node = {INTERFACE_NODE, "%s(config-if)# ", 1};
 
+#ifndef VTYSH_EXTRACT_PL
+#include "zebra/interface_clippy.c"
+#endif
 /* Show all interfaces to vty. */
-DEFUN (show_interface,
-       show_interface_cmd,
-       "show interface [vrf NAME]",
-       SHOW_STR
-       "Interface status and configuration\n"
-       VRF_CMD_HELP_STR)
+DEFPY(show_interface, show_interface_cmd,
+      "show interface [vrf NAME$name] [brief$brief]",
+      SHOW_STR "Interface status and configuration\n" VRF_CMD_HELP_STR
+	       "Interface status and configuration summary\n")
```

Comment:
Please put the formatting back how it was, one thing per line. We use non-standardish formatting for DEFUNs, I know clang-format kind of munges these; in the future if you want to run clang-format over a big patch but not touch these I usually just put
```
/* clang-format off */
...
/* clang-format on */
```
around defuns

Reply:
Yupp, this is by clang-format. Done. Thanks, for the tip.